dns.opam

```diff
@@ -15,7 +15,6 @@ depends: [
   "gmap" {>= "0.3.0"}
   "cstruct" {>= "3.2.0"}
   "ipaddr" {>= "3.0.0"}
-  "ppx_expect"
```

Comment:
since it is not run in a with-test due to the issues of runtime dependencies, i prefer to not have it in opam, as done in here. i can see multiple paths forward: (a) get rid of the tests entirely (b) migrate them to the tests directory at the cost of exposing various types and modules and functions (c) wait until ppx_expect does not drag any dependencies along when not testing... i still have hope (c) will be done at some point.